app/src/main/res/drawable/baseline_battery_0_bar_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:height="24dp" android:tint="#000000" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp">
```

Comment:
I actually don't know if it makes a difference anymore, but I usually set the `android:tint` to `?attr/colorControlNormal`.

(Same in all the drawable)